assets/javascripts/discourse/components/modal/share-modal.gjs

```diff
@@ -104,6 +116,13 @@ export default class ShareModal extends Component {
           @label="discourse_ai.ai_bot.share_modal.copy"
         />
         <span class="ai-share-modal__just-copied">{{this.justCopiedText}}</span>
+        {{#if this.siteSettings.ai_bot_allow_public_sharing}}
```

Comment:
If the site setting becomes a group one this would be `this.currentUser.can_share_ai_bot_conversations`